data/mods/Magiclysm/Spells/technomancer.json

```diff
@@ -344,6 +344,32 @@
     "max_range": 30,
     "range_increment": 1
   },
+  {
+    "id": "summon_floating_disk",
+    "type": "SPELL",
+    "name": "Summon floating disk",
+    "description": "Summons a floating disk that is sworn to carry your burdens.",
+    "valid_targets": [ "ground" ],
+    "flags": [ "SOMATIC", "VERBAL", "CONCENTRATE" ],
+    "effect": "summon_vehicle",
+    "spell_class": "TECHNOMANCER",
+    "energy_source": "MANA",
+    "vehicle_id": "floating_disk",
```

Comment:
This vehicle is not defined anywhere.

Reply:
Oh woops, I forgot to add the new files I created, sorry. ill commit them now.